docs/developers/guides/token.md

```diff
@@ -0,0 +1,1287 @@
+ [Success]: ✔ should not mint tokens if not contract account RTrace: +207
+[Event] koinos.contracts.token.mint_event / [ '1DQzuCcTKacbs9GGScRTU1Hc8BsyARTPqK' ] / ChkAiCtotTmdMTyQZ/OEFgfmKMEtVCA8jEc6EHs=
+[Contract Exit] Mint would overflow supply
+ [Success]: ✔ should not mint tokens if new total supply overflows RTrace: +338
+[Event] koinos.contracts.token.mint_event / [ '1DQzuCcTKacbs9GGScRTU1Hc8BsyARTPqK' ] / ChkAiCtotTmdMTyQZ/OEFgfmKMEtVCA8jEc6EHs=
+[Contract Exit] Mint would overflow max supply
+ [Success]: ✔ should not mint tokens if new total supply overflows max supply RTrace: +338
+[Event] koinos.contracts.token.mint_event / [ '1DQzuCcTKacbs9GGScRTU1Hc8BsyARTPqG' ] / ChkAiCtotTmdMTyQZ/OEFgfmKMEtVCA8jEc3EHs=
+[Event] koinos.contracts.token.transfer_event / [
+  '1DQzuCcTKacbs9GGScRTU1Hc8BsyARTPqK',
+  '1DQzuCcTKacbs9GGScRTU1Hc8BsyARTPqG'
+] / ChkAiCtotTmdMTyQZ/OEFgfmKMEtVCA8jEc3EhkAiCtotTmdMTyQZ/OEFgfmKMEtVCA8jEc6GAo=
+ [Success]: ✔ should transfer tokens RTrace: +391
+[Event] koinos.contracts.token.mint_event / [ '1DQzuCcTKacbs9GGScRTU1Hc8BsyARTPqG' ] / ChkAiCtotTmdMTyQZ/OEFgfmKMEtVCA8jEc3EHs=
+[Contract Exit] 'from' has not authorized transfer
+ [Success]: ✔ should not transfer tokens without the proper authorizations RTrace: +295
+[Event] koinos.contracts.token.mint_event / [ '1DQzuCcTKacbs9GGScRTU1Hc8BsyARTPqG' ] / ChkAiCtotTmdMTyQZ/OEFgfmKMEtVCA8jEc3EHs=
+[Contract Exit] Cannot transfer to self
+ [Success]: ✔ should not transfer tokens to self RTrace: +234
+[Event] koinos.contracts.token.mint_event / [ '1DQzuCcTKacbs9GGScRTU1Hc8BsyARTPqG' ] / ChkAiCtotTmdMTyQZ/OEFgfmKMEtVCA8jEc3EHs=
+[Contract Exit] 'from' has insufficient balance
+ [Success]: ✔ should not transfer if insufficient balance RTrace: +307
+
+    [File]: assembly/__tests__/Token.spec.ts
+  [Groups]: 2 pass, 2 total
+  [Result]: ✔ PASS
+[Snapshot]: 0 total, 0 added, 0 removed, 0 different
+ [Summary]: 13 pass,  0 fail, 13 total
+    [Time]: 139.55ms
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  [Result]: ✔ PASS
+   [Files]: 1 total
+  [Groups]: 2 count, 2 pass
+   [Tests]: 13 pass, 0 fail, 13 total
+    [Time]: 5130.905ms
+┌───────────────────┬───────┬───────┬──────┬──────┬───────────┐
+│ File              │ Total │ Block │ Func │ Expr │ Uncovered │
+├───────────────────┼───────┼───────┼──────┼──────┼───────────┤
+│ assembly/Token.ts │ 100%  │ 100%  │ 100% │ 100% │           │
+├───────────────────┼───────┼───────┼──────┼──────┼───────────┤
+│ total             │ 100%  │ 100%  │ 100% │ 100% │           │
+└───────────────────┴───────┴───────┴──────┴──────┴───────────┘
+
+```
+
+---
+## Customizing the token
+Let's customize the specifics of our token project by modifying `assembly/Token.ts`. Define the following:
+
+- `_name`: The name of your token
+
+- `_symbol`: The symbol for your token
+
+- `_decimals`: The decimal places for your token
+
+
+```ts linenums="1" title="assembly/Token.ts"
```

Comment:
Let's show this change in context with the source file and highlight the modified lines.

Reply:
You did not make the update as I suggested, only matched the line numbers. I would add more context of the source file (3-4 lines on either side) and then highlight the modified lines.

Here is how you add highlighted lines.

https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#highlighting-specific-lines